app/javascript/mastodon/components/hover_card.tsx

```diff
@@ -0,0 +1,205 @@
+import { useEffect, useRef, useState, useCallback } from 'react';
+
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+
+import Overlay from 'react-overlays/Overlay';
+import type { OffsetValue } from 'react-overlays/esm/usePopper';
+
+import { fetchAccount } from 'mastodon/actions/accounts';
+import { AccountBio } from 'mastodon/components/account_bio';
+import { AccountFields } from 'mastodon/components/account_fields';
+import { Avatar } from 'mastodon/components/avatar';
+import { FollowersCounter } from 'mastodon/components/counters';
+import { DisplayName } from 'mastodon/components/display_name';
+import { FollowButton } from 'mastodon/components/follow_button';
+import { LoadingIndicator } from 'mastodon/components/loading_indicator';
+import { ShortNumber } from 'mastodon/components/short_number';
+import { domain } from 'mastodon/initial_state';
+import { useAppSelector, useAppDispatch } from 'mastodon/store';
+
+const offset = [-12, 4] as OffsetValue;
+const enterDelay = 250;
+const leaveDelay = 300;
```

Comment:
The 250 enter delay seems too slow to me. I failed to find any guidance on this but I experimented with a few values and ~600ms seems like the sweet spot to me.

The leave delay seems adequate, but there is a minor bug where hovering *any* hover-card-enabled link holds the first hover card open (until a new hover card opens), which can be surprising and jarring when you move to another link.

Reply:
Did you mean 250ms is too fast? I've changed this to 500ms now.

Reply:
500ms works a lot better, thanks!